src/main/javascript/view/OperationView.js

```diff
@@ -313,7 +313,7 @@ SwaggerUi.Views.OperationView = Backbone.View.extend({
       if (isFileUpload) {
         return this.handleFileUpload(map, form);
       } else {
-        return this.model['do'](map, opts, this.showCompleteStatus, this.showErrorStatus, this);
+        return this.model.execute(map, this.options.swaggerOptions, this.showCompleteStatus, this.showErrorStatus);
```

Comment:
This object ```OperationView``` is created here: https://github.com/swagger-api/swagger-ui/blob/develop_2.0/src/main/javascript/view/ResourceView.js#L52-L59 

Basically, ```this.options``` is Backbone.js stuff that gets passed around. Good idea to add the guard to empty {}.